mkosi.profiles/kde/mkosi.conf

```diff
@@ -4,6 +4,8 @@
 
 [Content]
 Packages=
+        flatpak
```

Comment:
Curious why you think flatpak would be KDE specific? It works on GNOME as well no?

Reply:
It's not KDE specific, but it seemed to me something more usecase specific than something we should have in a base profile (like, if you're making an appliance-style system you probably wouldn't want it). Debatable of course, like most things :)

Reply:
Good point, it doesn't really fit in with the rest of the packages in the desktop profile. We should probably split a graphics profile from the desktop profile with all the graphics stuff and then the desktop profile can have flatpak in it. For now feel free to drop flatpak, I can just add it to my local config.

Reply:
Dropped